packages/react/time-series/src/TimeSeries.tsx

```diff
@@ -203,6 +196,32 @@ export function TimeSeries(props: TimeSeriesProps) {
         }
       }
 
+      if (chartRef.current) {
+        const datasetRef = chartRef.current.data.datasets[0]
```

Comment:
is this really a `React.Ref`? I think it could be just `const dataset = ...`

Reply:
No, it's not a react ref, it's the reference to the dataset object in memory but I agree changing the name

Reply:
No, it's not a react ref, it's the reference to the dataset object in memory but I agree changing the name